lib/nxt_state_machine/state_machine.rb

```diff
@@ -75,12 +75,17 @@ def any_state
       states.values.map(&:enum)
     end
 
+    alias_method :all_states_except, :any_state_except
     alias_method :all_states, :any_state
 
     def all_states_except(*excluded)
       all_states - excluded
     end
 
+    def any_state_except(*excluded)
+      any_state - excluded
+    end
+
```

Comment:
`all_states_except` and `any_state_except` are basically the same method because of the `alias_method`.
This definition is not required.